src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,7 +35,15 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(trimmedAddress)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
-        this.value = trimmedAddress;
+        processAddress(trimmedAddress);
+    }
+
+    private void processAddress(String trimmedAddress) {
+        String[] result = trimmedAddress.split(",");
```

Comment:
The `COMMA` constant isn't used here. Is it only for converting `Address` into a string?